src/main/java/net/tirasa/connid/bundles/azure/AzureConnector.java

```diff
@@ -848,15 +849,36 @@ private ConnectorObject fromUser(final User user, final Set<String> attributesTo
                                         field.getName(), field.getType()).build());
                                 break;
                             case "provisionedPlans":
-                                attrs.add(AzureAttributes.doBuildAttributeFromClassField(user.provisionedPlans,
+                                attrs.add(AzureAttributes.doBuildAttributeFromClassField(
+                                        user.provisionedPlans != null
```

Comment:
1. invert conditions, e.g.

```
user.provisionedPlans == null ? null : etc
```

2. use method references e.g. replace

```
provisionedPlan -> provisionedPlan.service
```

with

```
ProvisionedPlan::service
```

Reply:
ok, to invert conditions, but for method references, unfortunately they are not method, but just attribute with public access modifier

Reply:
> but for method references, unfortunately they are not method, but just attribute with public access modifier

ouch, I see
ok, let's leave that as it is...